### PR TITLE
Add 2 new choti moti bat

### PR DESCRIPTION
**Pull request by sahilbathlaofficial**

### CODE REVIEW

File: _posts/2015-12-15-promise-again [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+layout: post
+title:  "Promise And Resolve"
+date:   2015-12-15 01:49:56
+categories: nodejs javascript
+author:
+ twitter: sahilbathla
+ name: Sahil Batla
+---
+
+Sirf "Promise" mat karo, Use "Resolve" bhi karo
```
Comment (AshBardhan) [inline _posts/2015-12-15-promise-again:11]: Use this for writing text in hindi
http://www.quillpad.in/index.html#.VmfyPBArKRs

File: _posts/2015-12-15-global-warming [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+layout: post
+title:  "Global Variables"
+date:   2015-12-15 01:49:56
+categories: nodejs javascript
+author:
+ twitter: sahilbathla
+ name: Sahil Batla
+---
+
+Ek acha Prani(Human) wahi hai jo "Global Warming" and "Global Variables" dono kum kare.
```
Comment (AshBardhan) [inline _posts/2015-12-15-global-warming:11]: `Insaan` sahi hai

File: _posts/2015-12-15-global-warming [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+layout: post
+title:  "Global Variables"
+date:   2015-12-15 01:49:56
+categories: nodejs javascript
+author:
+ twitter: sahilbathla
+ name: Sahil Batla
```
Comment (AshBardhan) [inline _posts/2015-12-15-global-warming:8]: You can add your name and twitter handle in the `authors` list present in the `config.yml` file
From there you can add them here with `authorslug`

Comment (chinchang) [inline _posts/2015-12-15-global-warming:8]: this is fine. Cconfig.yml will be deprecated.

**Comment by chinchang** [comment]
please open separate PR for every choti-moti baat from next time so each can be reviewed independently.

**Comment by chinchang** [comment]
@sahilbathlaofficial Lemme know if you are gonna convert them in hindi. Else +1 from my side.

**Comment by sahilbathlaofficial** [comment]
@chinchang  - Good to go, @ashish - will do it next time.

**Comment by AshBardhan** [comment]
@sahilbathlaofficial Roger that :)

**Comment by chinchang** [comment]
@sahilbathlaofficial no testing here also. :dizzy_face: File names are wrong.

**Pull request merged**